src/app/components/course-island/course-island.component.scss

```diff
@@ -11,3 +11,7 @@
         @include tui-space(bottom, 3);
     }
 }
+
+.add-color {
+    background-color: rgba(218,179,249,0.5);
```

Comment:
What is this color? We don't want to have hardcoded colors, use taiga-ui colors instead.

Reply:
rgba was the only way I could find to affect the opacity of a background colour without affecting the opacity of the text as well. If I use a standard tui colour it looks like this:
![image](https://user-images.githubusercontent.com/79422004/218387758-ad8b530e-deb0-4f08-87e2-1db489a85dde.png)
Which looks weird to me because the colour is not opaque like the 'success' 'error' and 'warning' colours are. Those are also the only coloured non-custom options for tui-tag in the documentation.